airflow/utils/logging.py

```diff
@@ -195,23 +181,15 @@ def write(self, log, remote_log_location, append=False):
                 log = old_log + '\n' + log
 
             try:
-                if self.use_gcloud:
-                    self.hook.upload_from_string(
-                        log,
-                        blob=remote_log_location,
-                        replace=True)
-                    return
-                else:
-                    bkt, blob = self.parse_gcs_url(remote_log_location)
-                    from tempfile import NamedTemporaryFile
-                    with NamedTemporaryFile(mode='w+') as tmpfile:
-                        tmpfile.write(log)
-                        # Force the file to be flushed, since we're doing the
-                        # upload from within the file context (it hasn't been
-                        # closed).
-                        tmpfile.flush()
-                        self.hook.upload(bkt, blob, tmpfile.name)
-                    return
+                bkt, blob = self.parse_gcs_url(remote_log_location)
+                from tempfile import NamedTemporaryFile
+                with NamedTemporaryFile(mode='w+') as tmpfile:
+                    tmpfile.write(log)
+                    # Force the file to be flushed, since we're doing the
+                    # upload from within the file context (it hasn't been
+                    # closed).
+                    tmpfile.flush()
+                    self.hook.upload(bkt, blob, tmpfile.name)
             except:
                 pass
```

Comment:
I think the `logging.error()` that is below this line (not shown in github diff) should now go inside this except statement. Otherwise it prints the error every time